packages/vx-xychart/src/context/DataContext.tsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import React from 'react';
+import { DataContext as DataContextType } from '../types/data';
+
+type AnyDataContext = DataContextType<any, any, any, any>;
+
+/** Utilities for infering context generics */
+export type InferXScaleConfig<X extends AnyDataContext> = X extends DataContextType<
+  infer T,
+  any,
+  any,
+  any
+>
+  ? T
+  : any;
```

Comment:
can change this `any` to default type for XScaleConfig. May be `AxisScaleConfig`?

Reply:
ohhh I see what you mean now, sorry I didn't get this yesterday 👌